src/helm/tokenizers/cohere_tokenizer.py

```diff
@@ -49,17 +36,15 @@ def _tokenize_do_it(self, request: Dict[str, Any]) -> Dict[str, Any]:
         }
         """
         text: str = request["text"]
-        assert (
-            1 <= len(text) <= CohereTokenizer.TOKENIZE_API_MAX_TEXT_LENGTH
-        ), f"Invalid text length: {len(text)}. Valid length: [1..{CohereTokenizer.TOKENIZE_API_MAX_TEXT_LENGTH:,d}]"
+        assert (1 <= len(text)), f"Invalid empty text with length: {len(text)}. Valid length: >= 1"
+        request["model"] = "command-r" ### To do
```

Comment:
You can read this from the request (and also, avoid mutating the input parameter):
```
if request["tokenizer"] == "cohere/command-r":
    request = {"model: "command-r", **request}
```